docker/intro/README.MD

```diff
@@ -0,0 +1,365 @@
+# Introduction to Fn with Docker
+Fn is a lightweight Docker-based serverless functions platform you can run on
+your laptop, server, or cloud.  In this introductory tutorial we'll see how to
+use an existing Docker image as an Fn function and deploy this function to a local
+Fn server. 
+
+### Before you Begin
+* Set aside about 10 minutes to complete this tutorial.
+* Make sure Fn server is up and running by completing the [Install and Start Fn Tutorial](../../install/README.md).
+
+> As you make your way through this tutorial, look out for this icon.
+![](../../images/userinput.png) Whenever you see it, it's time for you to
+perform an action.
+
+## Your First Function
+Now that Fn server is up and running, let's start with a simple "hello world" function 
+using the Docker [hello-world image](https://hub.docker.com/_/hello-world/).
+
+Let's set the `FN_REGISTRY` environment variable. Normally, it's set to your Docker Hub username.
+However in this tutorial we'll work in local development mode so we can set
+the `FN_REGISTRY` variable to an arbitrary value. Let's use `fndemouser`.
+
+![user input](../../images/userinput.png)
+>```
+> export FN_REGISTRY=fndemouser
+>```
+
+Let's create a `dockerfn` directory.
+
+![user input](../../images/userinput.png)
+>```
+> mkdir dockerfn
+>```
+
+Let's switch to this directory.
+
+![user input](../../images/userinput.png)
+>```
+> cd dockerfn
+>```
+
+Now let's create a Dockerfile. 
+
+![](../../images/userinput.png) Copy/paste the following into a file named `Dockerfile`:
+
+```dockerfile
+FROM hello-world
```

Comment:
Probably need to explain what `hello-world` is and why this can work as a function.  I can see the attraction of using an off the shelf image but the mechanics may not be as clear as in the Node.js based Dockerfile hello world in the [Container as Function](https://github.com/fnproject/tutorials/tree/master/ContainerAsFunction) tutorial.